kolibri/content/api.py

```diff
@@ -0,0 +1,222 @@
+"""
+This module acts as the only interface point between other apps and the database backend for the content.
+It exposes several convenience functions for accessing content
+"""
+from kolibri.content import models as KolibriContent
+from uuid import UUID
+import os
+from bulk_update.helper import bulk_update
+from django.core.files import File as DjFile
+
+"""helper funcitons"""
```

Comment:
you mean when other app use this helper function, that app doesn't need to import every module in content app. But it won't make a difference to content app not matter is_valid_uuid is in a utils module or not. Is this what you mean?